docs/backup_and_restore.rst

```diff
@@ -104,6 +104,27 @@ archive in the output of the backup command.
 
 .. include:: includes/backup-warning.txt
 
+.. _backing_up_logos:
+
+Backing up your custom logo
+'''''''''''''''''''''''''''
+
+As of SecureDrop |version|, the backup will not include any custom logo file
```

Comment:
This behaviour's been the same since the custom logo settings moved into the JI, by my casual read, so I think you can leave out the reference to the version (it might be misleading to folks who have an older backup and expect that it will get restored)

Reply:
Good point, removed in beef2f0da18fa6276631112d862d4afe925a817f, we'll just remove that language in its entirety once the issue is fixed.